src/protocol/templates/ping_protocol.json

```diff
@@ -446,13 +446,24 @@
                     "description": "Temperature of the on-board thermistor.",
                     "payload": [
                         {
-                            "name": "temp",
+                            "name": "pcb_temperature",
```

Comment:
Modifications in ping-viewer will be necessary.

Reply:
Yes I will address after approval here, before merge